pipelines/testing/ImputationBeagleEmpty.wdl

```diff
@@ -13,7 +13,7 @@ workflow ImputationBeagle {
 
         File ref_dict # for reheadering / adding contig lengths in the header of the output VCF, and calculating contig lengths
         Array[String] contigs
-        String reference_panel_path # path to the bucket where the reference panel files are stored for all contigs
+        String reference_panel_path_prefix # path to the bucket where the reference panel files are stored for all contigs
```

Comment:
super minor: should we update the comment here, or just delete all the comments and leave the explanations on the main wdl?

Reply:
good point, i think i'll just delete all the comments